jsk_perception/src/draw_labeled_rect.cpp

```diff
@@ -0,0 +1,147 @@
+/*********************************************************************
```

Comment:
@wkentaro 

> Why not name it as `draw_rects.cpp` or `draw_rect_array.cpp` with subscription of `~input/rect_array` and `~input/labeled_rect_array`, with adjustments of code for both `RectArray` and `LabeledRectArray`?

OK. updated and now node is `draw_rects.cpp` which subscribes `RectArray` and `LabeledRectArray`